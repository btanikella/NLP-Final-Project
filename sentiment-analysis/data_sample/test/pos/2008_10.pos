DT JJ NN VBZ DT NN IN VBG PRP$ NN RB IN JJ NN NN CC JJ CC JJ NNS
DT NN RB IN VBG NN IN JJ NN VBZ RB JJ CC DT NN VBZ RB JJ NN TO VB DT NN IN CD NNS NN VBZ DT NN IN DT NN TO VB DT NN JJ RB IN DT NN TO VB IN VBG DT VBG NN
PRP RB VBP DT NN NNS VBD DT NN IN NN TO VBG PRP$ NNS VBD IN PRP$ NNP VBD
