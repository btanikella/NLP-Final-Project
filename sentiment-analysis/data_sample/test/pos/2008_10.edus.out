Sentence #1 (21 tokens):
This young filmmaker has a talent for capturing his audience quickly with unusual camera work and sparse but intense scripts.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=young CharacterOffsetBegin=9 CharacterOffsetEnd=14 PartOfSpeech=JJ] [Text=filmmaker CharacterOffsetBegin=15 CharacterOffsetEnd=24 PartOfSpeech=NN] [Text=has CharacterOffsetBegin=25 CharacterOffsetEnd=28 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=29 CharacterOffsetEnd=30 PartOfSpeech=DT] [Text=talent CharacterOffsetBegin=31 CharacterOffsetEnd=37 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=38 CharacterOffsetEnd=41 PartOfSpeech=IN] [Text=capturing CharacterOffsetBegin=42 CharacterOffsetEnd=51 PartOfSpeech=VBG] [Text=his CharacterOffsetBegin=52 CharacterOffsetEnd=55 PartOfSpeech=PRP$] [Text=audience CharacterOffsetBegin=56 CharacterOffsetEnd=64 PartOfSpeech=NN] [Text=quickly CharacterOffsetBegin=65 CharacterOffsetEnd=72 PartOfSpeech=RB] [Text=with CharacterOffsetBegin=73 CharacterOffsetEnd=77 PartOfSpeech=IN] [Text=unusual CharacterOffsetBegin=78 CharacterOffsetEnd=85 PartOfSpeech=JJ] [Text=camera CharacterOffsetBegin=86 CharacterOffsetEnd=92 PartOfSpeech=NN] [Text=work CharacterOffsetBegin=93 CharacterOffsetEnd=97 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=98 CharacterOffsetEnd=101 PartOfSpeech=CC] [Text=sparse CharacterOffsetBegin=102 CharacterOffsetEnd=108 PartOfSpeech=JJ] [Text=but CharacterOffsetBegin=109 CharacterOffsetEnd=112 PartOfSpeech=CC] [Text=intense CharacterOffsetBegin=113 CharacterOffsetEnd=120 PartOfSpeech=JJ] [Text=scripts CharacterOffsetBegin=121 CharacterOffsetEnd=128 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=128 CharacterOffsetEnd=129 PartOfSpeech=.] 
root(ROOT-0, has-4)
det(filmmaker-3, This-1)
amod(filmmaker-3, young-2)
nsubj(has-4, filmmaker-3)
det(talent-6, a-5)
dobj(has-4, talent-6)
prep(talent-6, for-7)
pobj(for-7, capturing-8)
poss(audience-10, his-9)
dobj(capturing-8, audience-10)
advmod(capturing-8, quickly-11)
prep(capturing-8, with-12)
amod(work-15, unusual-13)
nn(work-15, camera-14)
pobj(with-12, work-15)
cc(work-15, and-16)
conj(work-15, sparse-17)
cc(capturing-8, but-18)
amod(scripts-20, intense-19)
conj(capturing-8, scripts-20)
punct(has-4, .-21)

Sentence #2 (50 tokens):
The concept here of combining animation with live footage is remarkably well-executed and the soundtrack is very good.The decision to release the movie in twelve parts online puts the onus on the director to make each episode fascinating enough for the viewer to invest in buying each upcoming episode.
[Text=The CharacterOffsetBegin=134 CharacterOffsetEnd=137 PartOfSpeech=DT] [Text=concept CharacterOffsetBegin=138 CharacterOffsetEnd=145 PartOfSpeech=NN] [Text=here CharacterOffsetBegin=146 CharacterOffsetEnd=150 PartOfSpeech=RB] [Text=of CharacterOffsetBegin=151 CharacterOffsetEnd=153 PartOfSpeech=IN] [Text=combining CharacterOffsetBegin=154 CharacterOffsetEnd=163 PartOfSpeech=VBG] [Text=animation CharacterOffsetBegin=164 CharacterOffsetEnd=173 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=174 CharacterOffsetEnd=178 PartOfSpeech=IN] [Text=live CharacterOffsetBegin=179 CharacterOffsetEnd=183 PartOfSpeech=JJ] [Text=footage CharacterOffsetBegin=184 CharacterOffsetEnd=191 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=192 CharacterOffsetEnd=194 PartOfSpeech=VBZ] [Text=remarkably CharacterOffsetBegin=195 CharacterOffsetEnd=205 PartOfSpeech=RB] [Text=well-executed CharacterOffsetBegin=206 CharacterOffsetEnd=219 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=220 CharacterOffsetEnd=223 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=224 CharacterOffsetEnd=227 PartOfSpeech=DT] [Text=soundtrack CharacterOffsetBegin=228 CharacterOffsetEnd=238 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=239 CharacterOffsetEnd=241 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=242 CharacterOffsetEnd=246 PartOfSpeech=RB] [Text=good.The CharacterOffsetBegin=247 CharacterOffsetEnd=255 PartOfSpeech=JJ] [Text=decision CharacterOffsetBegin=256 CharacterOffsetEnd=264 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=265 CharacterOffsetEnd=267 PartOfSpeech=TO] [Text=release CharacterOffsetBegin=268 CharacterOffsetEnd=275 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=276 CharacterOffsetEnd=279 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=280 CharacterOffsetEnd=285 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=286 CharacterOffsetEnd=288 PartOfSpeech=IN] [Text=twelve CharacterOffsetBegin=289 CharacterOffsetEnd=295 PartOfSpeech=CD] [Text=parts CharacterOffsetBegin=296 CharacterOffsetEnd=301 PartOfSpeech=NNS] [Text=online CharacterOffsetBegin=302 CharacterOffsetEnd=308 PartOfSpeech=NN] [Text=puts CharacterOffsetBegin=309 CharacterOffsetEnd=313 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=314 CharacterOffsetEnd=317 PartOfSpeech=DT] [Text=onus CharacterOffsetBegin=318 CharacterOffsetEnd=322 PartOfSpeech=NN] [Text=on CharacterOffsetBegin=323 CharacterOffsetEnd=325 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=326 CharacterOffsetEnd=329 PartOfSpeech=DT] [Text=director CharacterOffsetBegin=330 CharacterOffsetEnd=338 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=339 CharacterOffsetEnd=341 PartOfSpeech=TO] [Text=make CharacterOffsetBegin=342 CharacterOffsetEnd=346 PartOfSpeech=VB] [Text=each CharacterOffsetBegin=347 CharacterOffsetEnd=351 PartOfSpeech=DT] [Text=episode CharacterOffsetBegin=352 CharacterOffsetEnd=359 PartOfSpeech=NN] [Text=fascinating CharacterOffsetBegin=360 CharacterOffsetEnd=371 PartOfSpeech=JJ] [Text=enough CharacterOffsetBegin=372 CharacterOffsetEnd=378 PartOfSpeech=RB] [Text=for CharacterOffsetBegin=379 CharacterOffsetEnd=382 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=383 CharacterOffsetEnd=386 PartOfSpeech=DT] [Text=viewer CharacterOffsetBegin=387 CharacterOffsetEnd=393 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=394 CharacterOffsetEnd=396 PartOfSpeech=TO] [Text=invest CharacterOffsetBegin=397 CharacterOffsetEnd=403 PartOfSpeech=VB] [Text=in CharacterOffsetBegin=404 CharacterOffsetEnd=406 PartOfSpeech=IN] [Text=buying CharacterOffsetBegin=407 CharacterOffsetEnd=413 PartOfSpeech=VBG] [Text=each CharacterOffsetBegin=414 CharacterOffsetEnd=418 PartOfSpeech=DT] [Text=upcoming CharacterOffsetBegin=419 CharacterOffsetEnd=427 PartOfSpeech=VBG] [Text=episode CharacterOffsetBegin=428 CharacterOffsetEnd=435 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=435 CharacterOffsetEnd=436 PartOfSpeech=.] 
root(ROOT-0, well-executed-12)
det(concept-2, The-1)
nsubj(well-executed-12, concept-2)
advmod(concept-2, here-3)
prep(concept-2, of-4)
pcomp(of-4, combining-5)
dobj(combining-5, animation-6)
prep(combining-5, with-7)
amod(footage-9, live-8)
pobj(with-7, footage-9)
cop(well-executed-12, is-10)
advmod(well-executed-12, remarkably-11)
cc(well-executed-12, and-13)
det(soundtrack-15, the-14)
nsubj(decision-19, soundtrack-15)
cop(decision-19, is-16)
advmod(good.The-18, very-17)
amod(decision-19, good.The-18)
conj(well-executed-12, decision-19)
aux(release-21, to-20)
infmod(decision-19, release-21)
det(movie-23, the-22)
dobj(release-21, movie-23)
prep(release-21, in-24)
num(parts-26, twelve-25)
pobj(in-24, parts-26)
nsubj(puts-28, online-27)
rcmod(parts-26, puts-28)
det(onus-30, the-29)
dobj(puts-28, onus-30)
prep(puts-28, on-31)
det(director-33, the-32)
pobj(on-31, director-33)
aux(make-35, to-34)
xcomp(release-21, make-35)
det(episode-37, each-36)
nsubj(fascinating-38, episode-37)
xcomp(make-35, fascinating-38)
advmod(invest-44, enough-39)
mark(invest-44, for-40)
det(viewer-42, the-41)
nsubj(invest-44, viewer-42)
aux(invest-44, to-43)
ccomp(fascinating-38, invest-44)
prep(invest-44, in-45)
pcomp(in-45, buying-46)
det(episode-49, each-47)
nn(episode-49, upcoming-48)
dobj(buying-46, episode-49)
punct(well-executed-12, .-50)

Sentence #3 (21 tokens):
I only wish all motion pictures had this kind of commitment to keeping their audiences entertained throughout their stories.Highly recommended.
[Text=I CharacterOffsetBegin=441 CharacterOffsetEnd=442 PartOfSpeech=PRP] [Text=only CharacterOffsetBegin=443 CharacterOffsetEnd=447 PartOfSpeech=RB] [Text=wish CharacterOffsetBegin=448 CharacterOffsetEnd=452 PartOfSpeech=VBP] [Text=all CharacterOffsetBegin=453 CharacterOffsetEnd=456 PartOfSpeech=DT] [Text=motion CharacterOffsetBegin=457 CharacterOffsetEnd=463 PartOfSpeech=NN] [Text=pictures CharacterOffsetBegin=464 CharacterOffsetEnd=472 PartOfSpeech=NNS] [Text=had CharacterOffsetBegin=473 CharacterOffsetEnd=476 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=477 CharacterOffsetEnd=481 PartOfSpeech=DT] [Text=kind CharacterOffsetBegin=482 CharacterOffsetEnd=486 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=487 CharacterOffsetEnd=489 PartOfSpeech=IN] [Text=commitment CharacterOffsetBegin=490 CharacterOffsetEnd=500 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=501 CharacterOffsetEnd=503 PartOfSpeech=TO] [Text=keeping CharacterOffsetBegin=504 CharacterOffsetEnd=511 PartOfSpeech=VBG] [Text=their CharacterOffsetBegin=512 CharacterOffsetEnd=517 PartOfSpeech=PRP$] [Text=audiences CharacterOffsetBegin=518 CharacterOffsetEnd=527 PartOfSpeech=NNS] [Text=entertained CharacterOffsetBegin=528 CharacterOffsetEnd=539 PartOfSpeech=VBD] [Text=throughout CharacterOffsetBegin=540 CharacterOffsetEnd=550 PartOfSpeech=IN] [Text=their CharacterOffsetBegin=551 CharacterOffsetEnd=556 PartOfSpeech=PRP$] [Text=stories.Highly CharacterOffsetBegin=557 CharacterOffsetEnd=571 PartOfSpeech=NNP] [Text=recommended CharacterOffsetBegin=572 CharacterOffsetEnd=583 PartOfSpeech=VBD] [Text=. CharacterOffsetBegin=583 CharacterOffsetEnd=584 PartOfSpeech=.] 
root(ROOT-0, wish-3)
nsubj(wish-3, I-1)
advmod(wish-3, only-2)
det(pictures-6, all-4)
nn(pictures-6, motion-5)
nsubj(had-7, pictures-6)
ccomp(wish-3, had-7)
det(kind-9, this-8)
dobj(had-7, kind-9)
prep(kind-9, of-10)
pobj(of-10, commitment-11)
aux(keeping-13, to-12)
xcomp(had-7, keeping-13)
poss(audiences-15, their-14)
nsubj(entertained-16, audiences-15)
ccomp(keeping-13, entertained-16)
mark(recommended-20, throughout-17)
poss(stories.Highly-19, their-18)
nsubj(recommended-20, stories.Highly-19)
ccomp(entertained-16, recommended-20)
punct(wish-3, .-21)

